libraries/chain/include/graphene/chain/protocol/custom_authority.hpp

```diff
@@ -68,10 +67,21 @@ namespace graphene { namespace chain {
          uint32_t price_per_k_unit = 100; ///< units = valid seconds * items in auth * items in restrictions
       };
 
-      asset             fee;
-      account_id_type   account;
-      uint64_t          delta_units; // to calculate fee, it will be validated in evaluator
-                                     // Note: if start was in the past, when updating, used fee should be deducted
+      object_id_type                  custom_authority_to_update;
+      
+      asset                           fee;
+      account_id_type                 account;
+      
+      bool                            enabled;
+      time_point_sec                  valid_from;
+      time_point_sec                  valid_to;
+      unsigned_int                    operation_type;
```

Comment:
What is `operation_type` used for?

Reply:
Please do that according to abits recommendation in the other review

Reply:
Yes, I've done this that way. Changes are already in PR.